components/gameComponents/hand.js

```diff
@@ -16,6 +16,13 @@ const Hand = ({ cards, playerTurn, comboIsValid, requestCombo, currentTurnCombo,
   const [hasReset, resetCombo] = useState(false);
   const isMyTurn = playerTurn === 0;
 
+  // Update hand after changes.
+  // NOTE: Apparently this might not be best practice, and some people suggest handling all
+  // state changes from the parent only. But that seems like too much managmement in one place imo.
+  if(cards.length !== hand.length) {
+    setHand(cards);
+  }
```

Comment:
I believe best practice to handle  state change from parent, would be to use a **useEffect** hook to allow the hand component to watch for changes to the **cards** prop, and re-rendering when **cards** changes.

https://react.dev/reference/react/useEffect
--> See Examples of Passing reactive dependencies

`useEffect(() => {
    setHand(cards);
  }, [cards])`

Reply:
ahh yes, that's probably the right way to handle this